benchmarks/lib/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 val accpVersion: String? by project
 val accpLocalJar: String by project
 val fips: Boolean by project
+val ib: String by project
```

Comment:
nit: The name is ambiguous. Maybe give it a long name instead? I think Gradle is smart enough to shorten a long name automatically so we can have it both ways.